go/integration_test/processor/calls_test.go

```diff
@@ -396,6 +397,66 @@ func TestProcessor_RecursiveCallsAfterAStaticCallAreStatic(t *testing.T) {
 	}
 }
 
+func TestProcessor_CallingNonExistentAccountFails(t *testing.T) {
+	tests := map[string]struct {
+		account            Account
+		callValue          tosca.Value
+		accountExistsAfter bool
+	}{
+		"empty": {
+			account:            Account{},
+			callValue:          tosca.Value{},
+			accountExistsAfter: false,
+		},
+		"callWithValue": {
+			account:            Account{},
+			callValue:          tosca.NewValue(42),
+			accountExistsAfter: true,
+		},
+		"existingAccount": {
+			account:            Account{Nonce: 1},
+			callValue:          tosca.Value{},
+			accountExistsAfter: true,
+		},
+	}
+	for processorName, processor := range getProcessors() {
+		if strings.Contains(processorName, "floria") {
+			continue
+		}
+		for testName, test := range tests {
+			t.Run(processorName+"/"+testName, func(t *testing.T) {
+				sender := tosca.Address{1}
+				receiver := tosca.Address{2}
+
+				state := WorldState{
+					sender:   Account{Balance: tosca.NewValue(100)},
+					receiver: test.account,
+				}
```

Comment:
for some reason this was confusing to read. I originally thought `sender` and `reciever` were fields that were getting assigned the other values. 
```suggestion
				state := WorldState{}
				state[sender] = Account{Balance: tosca.NewValue(100)}
				state[receiver] = test.account
```
maybe this is a more clear way of showing what the test is doing.

Reply:
I don't think this is readable, since it just repeats the name of the constructed object. I think it is very common in golang to directly fill maps with values.